buildroot/share/PlatformIO/scripts/STM32F1_create_variant.py

```diff
@@ -3,28 +3,40 @@
 #
 import pioutil
 if pioutil.is_pio_build():
-    import shutil, marlin
     from pathlib import Path
-    env = pioutil.env
-    platform = env.PioPlatform()
-    board = env.BoardConfig()
-
-    FRAMEWORK_DIR = Path(platform.get_package_dir("framework-arduinoststm32-maple"))
-    assert FRAMEWORK_DIR.is_dir()
 
-    source_root = Path("buildroot/share/PlatformIO/variants")
+    source_root_str = "buildroot/share/PlatformIO/variants"
+    source_root = Path(source_root_str)
     assert source_root.is_dir()
 
+    env = pioutil.env
+    board = env.BoardConfig()
     variant = board.get("build.variant")
-    variant_dir = FRAMEWORK_DIR / "STM32F1/variants" / variant
 
     source_dir = source_root / variant
     assert source_dir.is_dir()
 
-    if variant_dir.is_dir():
-        shutil.rmtree(variant_dir)
+    if True:
```

Comment:
?

Reply:
Developers always leaving placeholders, amirite?

Reply:
fair enough, but i didnt expect it to be actually merged to main ;)